lib/hanami/logger.rb

```diff
@@ -453,7 +454,14 @@ def _format(hash)
     #   logger.info "Hello World"
     #
     #   # => {"app":"Hanami","severity":"DEBUG","time":"2017-03-30T13:57:59Z","message":"Hello World"}
-    def initialize(application_name = nil, *args, stream: $stdout, level: DEBUG, formatter: nil, filter: []) # rubocop:disable Metrics/ParameterLists
+    # rubocop:disable Lint/HandleExceptions
+    # rubocop:disable Metrics/ParameterLists
+    def initialize(application_name = nil, *args, stream: $stdout, level: DEBUG, formatter: nil, filter: [])
+      begin
+        Utils::Files.mkdir_p(stream)
+      rescue TypeError
```

Comment:
@AlfonsoUceda Are we trying to create the directory regardless of the `stream` type?

Reply:
Yep, but I think that the reason of rescue